ynput/ayon-substance-painter
# improve the channel map filtering

## Changelog Description
The channel map export in the latest develop cannot filter some maps when the maps include similar names (i.e. Color and EmissionColor). This PR is to make sure the current channel map filtering would filter the maps as expected. 
![image](https://github.com/user-attachments/assets/3f6da9ec-21c1-4249-a076-ca8ecfd38ebf)


## Additional info
n/a

## Testing notes:

1. Launch Substance Painter with your project
2. Create Texture Set
3. Use RsMaterialTemplate and add base color. Following the setting as the screenshots showed. "Channel Type should be `_Color` for BaseColor if you are using RsMaterialTemplate"
![image](https://github.com/user-attachments/assets/b987f78f-0f68-432c-847d-83814aa95839)
![image](https://github.com/user-attachments/assets/ca12356e-7586-4045-a2cf-0ec7f33d35d9)

4. Publish
5. It should only publish Color Map.
